libde265.yaml

```diff
@@ -0,0 +1,62 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libde265/APKBUILD
```

Comment:
you wrote a tool to parse APKBUILD?? If you do, please do share it. I'm willing to be an alpha tester :D

Reply:
@rawlingsj did! We're still testing it but it works pretty well :) He's going to get it cleaned up and released soon :)

Reply:
Haha i planned to build it too but never gotten around to it

Reply:
Amazing works btw @rawlingsj